Formula/libstfl.rb

```diff
@@ -1,201 +1,77 @@
 class Libstfl < Formula
   desc "Library implementing a curses-based widget set for terminals"
   homepage "http://www.clifford.at/stfl/"
-  url "http://www.clifford.at/stfl/stfl-0.22.tar.gz"
-  sha256 "2dddafd331e61ad04baf1251ccecb3a69feaf99880dd75155be8423c1cc55418"
+  url "http://www.clifford.at/stfl/stfl-0.24.tar.gz"
+  sha256 "d4a7aa181a475aaf8a8914a8ccb2a7ff28919d4c8c0f8a061e17a0c36869c090"
 
   bottle do
     cellar :any
     revision 1
-    sha256 "98c3c02654f076427f5243913d027029cd4607642d3c212833c21c97d0093ea0" => :el_capitan
     sha256 "8ca63ef8c7776df77eda7a327b2e4a9d413992c08605e7922a7b82f560be618a" => :yosemite
     sha256 "9a0b5e72c534ca6bc3e1ba2fbafb384719e5df9663ff4b507936afebfde16272" => :mavericks
     sha256 "f30b9c76dbd4d31b7860b8ed8c0c8024f820624880db32b00d1804d38e9e6808" => :mountain_lion
   end
 
-  depends_on :python => :optional
-  depends_on "swig" => :build
+  option "without-perl", "Build without Perl support"
 
-  patch :DATA
+  depends_on :python => :recommended
```

Comment:
Or just remove this; it looks like even if I say --without-python, nothing gets passed that tells the build system to not build the Python stuff